dashboard/db/schema.rb

```diff
@@ -1981,12 +1982,6 @@
     t.index ["user_id"], name: "index_user_proficiencies_on_user_id", unique: true
   end
 
-  create_table "user_project_storage_ids", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
-    t.integer "user_id"
-    t.index ["user_id"], name: "user_id", unique: true
-    t.index ["user_id"], name: "user_storage_ids_user_id_index"
-  end
-
```

Comment:
No idea what happened here. Running `rake build` to see if that solves this one.

Reply:
Hmm, not sure what happened with this one. I ran `rake build` and it's still showing this. Knowing that `schema.rb` gets rebuilt, not sure how much this matters?

Reply:
you have to `cd pegasus; bundle exec rake db:migrate`. search slack for user_project_storage_ids to learn more

Reply:
apologies - per aforementioned slack convos, `rake build` should fix.

Reply:
Hmm those commands didn't work for me? I'll search on Slack

Reply:
not totally sure about the ordering, but after doing either of those, you might need to `cd dashboard ; rake db:migrate` again to confirm that it is no longer making changes to schema.yml.

Reply:
Brilliant! That worked